### DISCUSSION TITLE
Is it possible to have full capacity for an event?

### DISCUSSION THREAD

**Discussion by WaloGomez** [pretix/pretix#4632] [Q&A] [answered]
Hello everyone !
Congratulations on your great work! 
I hope someone can help me.
I can't find how to specify the total capacity for an event that has the following characteristic.
The event can be purchased by residents and non-residents and has different prices. The total capacity is 100 regardless of the number of residents or non-residents. 
E.g. 80 residents and 20 non-residents or 100 residents and 0 non-residents, etc.
I see that I can specify quotas for residents and non-residents that would be 100 for each, but the reality is that the total number of tickets purchased cannot exceed 100 regardless of whether they are residents or non-residents.
Is there a way to specify this? 
Thank you so much
Walter

**Comment by felixrindt** [comment, marked as answer]
You just need one quota ("Participants") and add both products to that quota.

**Comment by WaloGomez** [comment]
Thank very much !!!
